src/c-opentimelineio/opentime-bindings/include/copentime/errorStatus.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include <stdbool.h>
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+    struct ErrorStatus;
+    typedef struct ErrorStatus ErrorStatus;
+
+    typedef enum
+    {
+        OK                                   = 0,
+        INVALID_TIMECODE_RATE                = 1,
+        NON_DROPFRAME_RATE                   = 2,
+        INVALID_TIMECODE_STRING              = 3,
+        INVALID_TIME_STRING                  = 4,
+        TIMECODE_RATE_MISMATCH               = 5,
+        NEGATIVE_VALUE                       = 6,
+        INVALID_RATE_FOR_DROP_FRAME_TIMECODE = 7,
+    } Outcome_;
+    typedef int  Outcome;
```

Comment:
Ok, so unfortunately because C doesn't have namespaces, we should include them in both the enum type name and the enumerations.   Can we use this pattern for the other enums as well?

```suggestion
    typedef enum
    {
        OTIO_ErrorStatus_Outcome_OK                                   = 0,
        OTIO_ErrorStatus_Outcome_INVALID_TIMECODE_RATE                = 1,
        OTIO_ErrorStatus_Outcome_NON_DROPFRAME_RATE                   = 2,
        OTIO_ErrorStatus_Outcome_INVALID_TIMECODE_STRING              = 3,
        OTIO_ErrorStatus_Outcome_INVALID_TIME_STRING                  = 4,
        OTIO_ErrorStatus_Outcome_TIMECODE_RATE_MISMATCH               = 5,
        OTIO_ErrorStatus_Outcome_NEGATIVE_VALUE                       = 6,
        OTIO_ErrorStatus_Outcome_INVALID_RATE_FOR_DROP_FRAME_TIMECODE = 7,
    } OTIO_ErrorStatus_Outcome;
```

Reply:
Applied change